src/Compilers/CSharp/Test/Emit/Emit/EmitMetadata.cs

```diff
@@ -2149,7 +2149,8 @@ private void CheckInternalMembers(NamedTypeSymbol type, bool isFromSource)
                 Assert.Null(member);
         }
 
-        [Fact(Skip = "https://github.com/dotnet/roslyn/issues/6190"), WorkItem(90, "https://github.com/dotnet/roslyn/issues/90")]
+        [WorkItem(90, "https://github.com/dotnet/roslyn/issues/90")]
+        [Fact]
         public void EmitWithNoResourcesAllPlatforms()
```

Comment:
I see. It never failed for me so I did not know what could go wrong.
I hope the fix is sufficient.